pyproject.toml

```diff
@@ -87,6 +88,8 @@ select = ["ALL"]
 ignore = [
     "ANN002", # Missing type annotation for `*args`
     "ANN003", # Missing type annotation for `**kwargs`
+    "B905", # `strict` in zip
```

Comment:
Kill these. This is repeated from below i.e. lines 93 and 94 have the same ignore codes.